Include/cpython/monitoring.h

```diff
@@ -0,0 +1,163 @@
+#ifndef Py_CPYTHON_MONITORING_H
+#  error "this header file must not be included directly"
+#endif
+
+PyAPI_FUNC(void)
+PyMonitoring_EnterScope(PyMonitoringState *state_array, uint64_t *version,
+                        const uint8_t *event_types, uint32_t length);
+
+void
+_PyMonitoring_EnterScope(PyMonitoringState *state_array, uint64_t *version,
+                         const uint8_t *event_types, uint32_t length);
+
+PyAPI_FUNC(void)
+PyMonitoring_ExitScope(void);
+
+void
+_PyMonitoring_ExitScope(void);
+
+PyAPI_FUNC(int)
+PyMonitoring_FirePyStartEvent(PyMonitoringState *state, PyObject *codelike, int offset);
+
+int
+_PyMonitoring_FirePyStartEvent(PyMonitoringState *state, PyObject *codelike, int offset);
+
+PyAPI_FUNC(int)
+PyMonitoring_FirePyResumeEvent(PyMonitoringState *state, PyObject *codelike, int offset);
```

Comment:
This is declared in `Include/monitoring.h`, it shouldn't need to be declared here as well.

The same for the other functions.